### DISCUSSION TITLE
HELPPPPP

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2351] [Help with using Postal]
hello, i have an issue which i would love you to take a look into, i have my smtp configured on my which works fine but i have an issue especially when it comes to sending email to certain ISP for example, when i send email to @txt.att.net which is at&t carrier, i get an error response which is in the attachment below, also when i send to @tmomail.net and @vtext.com which are tmobile and verizon carrier respectively, my ip address gets blocked after sending just 2 emails even when the content says just HELLO. I need to have this fixed so badly, i wouldnt mind paying for consultation just to have this fixed please.
![attt](https://user-images.githubusercontent.com/96216385/217216588-5a2dd619-8dbc-4152-9c64-cee9a632e66e.png)

**Comment by willpower232** [comment]
The error message is referring to the return path which seems to be right however it seems you might not have port 25 open so it may be that the providers you're having an issue with are not able to validate the return path will work (as the port may not be open) so are rejecting your message?

Have you tried using mail-tester.com to verify everything is okay with your setup?

**Reply by Itseric100** [reply]
yes i tried mail tester and it gave me 10/10, it is just surprising i have to deal with this

**Reply by willpower232** [reply]
As I said, you don't seem to have port 25 open so that is probably the issue. Servers that receive email sometimes verify that the return path address is working and, in your case, it won't be if your Postal cannot receive email.

I guess mail-tester.com does not verify that so a bit unfortunate in this case.
